boilerplate/scripts/build.js

```diff
@@ -1,8 +1,8 @@
 import builder from "statinamic/lib/builder"
 
-import * as layouts from "../web_modules/layouts"
-import metadata from "../web_modules/app/metadata"
-import routes from "../web_modules/app/routes"
+const layouts = require.resolve("../web_modules/layouts")
+const metadata = require.resolve("../web_modules/app/metadata")
+const routes = require.resolve("../web_modules/app/routes")
```

Comment:
Maybe we can move this 3 lines in an "web_modules/app/exports.js" and reuse it in the index-client.js ?